samples/word/40-tables/manage-formatting.yaml

```diff
@@ -0,0 +1,281 @@
+order: 2
+id: word-tables-manage-formatting
+name: Get the formatting of a table
```

Comment:
Consider renaming this "Table formatting".

Reply:
done